docs/arch/adr-51.md

```diff
@@ -0,0 +1,37 @@
+# ADR 51: User Group Stats Page
+
+September 2023
+
+## Context
+
+We will create a user group stats page to provide statistical insights and data visualization of a user group’s Zooniverse activity. The user group stats page will utilize the new Zooniverse stats service - [ERAS](https://github.com/zooniverse/eras) (Enhanced Running Average Stats Service) - in addition to existing [Panoptes](https://github.com/zooniverse/panoptes) resources (notably `users`, `user_groups`, and `projects`).
+
+### Features
+
+The user group stats page will show statistical information based on user group member role and user group `stats_visibility` property.
+
+Statistical information will include a bar chart of classification counts or hours - changeable by period and filterable by project, top projects, and could include top individual contributors and all user group members’ statistics.
```

Comment:
I initially read this sentence as if the bar chart is filterable by top projects too so just a NB suggestion to phrase it as "a bar chart of classification counts or hours (changeable by period and filterable by project,) top projects, etc".